openverse_api/catalog/api/serializers/audio_serializers.py

```diff
@@ -211,7 +211,9 @@ def create(self, validated_data):
 
 class AudioWaveformSerializer(serializers.Serializer):
     len = serializers.SerializerMethodField()
-    points = serializers.ListField(serializers.FloatField(min_value=0, max_value=1))
+    points = serializers.ListField(
+        child=serializers.FloatField(min_value=0, max_value=1)
+    )
```

Comment:
I thought Black was configured to break lines at 88 characters. I might be wrong.